### PR TITLE
twitter login functionality and testing- Ashley and Gerardo

### PR DESCRIPTION
**Pull request by ashleyajen**
@orangewolf ready to get pulled

### CODE REVIEW

File: config/initializers/session_store.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_HoppyCrawl_session'
+Rails.application.config.session_store :active_record_store, key: '_HoppyCrawl_session'
```
Comment (orangewolf) [inline config/initializers/session_store.rb]: Is there a reason to change from cookie store to active_record store here?

File: spec/features/log_outs_spec.rb [hunk 2]
```diff
@@ -1,27 +0,0 @@
-require 'rails_helper'
-
-RSpec.feature "LogOuts", type: :feature do
```
Comment (orangewolf) [inline spec/features/log_outs_spec.rb]: what did the logout spec ever do to you guys?

Comment (orangewolf) [inline spec/features/log_outs_spec.rb]: aka - why did you delete it

Comment (ashleyajen) [inline spec/features/log_outs_spec.rb]: @orangewolf added the content back to the logout spec file
